js_task1/thirdTask.js

```diff
@@ -0,0 +1,151 @@
+const input = document.querySelector("input");
+const getMoneyBtn = document.querySelector("button");
+const infoBlock = document.querySelector(".info");
+
+class ATM {
+  userValue = 0;
+  maxStoreValue = 0;
+  moneyToOut = 0;
+  outNotes = [];
+  // set random counts of notes
+  store = [
+    { value: 1000, count: randomValue(5, 12) },
+    { value: 500, count: randomValue(9, 17) },
+    { value: 200, count: randomValue(20, 25) },
+    { value: 100, count: randomValue(20, 30) },
+    { value: 50, count: randomValue(40, 60) },
+    { value: 20, count: randomValue(80, 120) },
+    { value: 10, count: randomValue(100, 120) },
+    { value: 5, count: randomValue(140, 170) },
+    { value: 2, count: randomValue(150, 200) },
+    { value: 1, count: randomValue(200, 220) },
+  ];
+
+  constructor() {
+    this.calculateStoreMoney(); //first time calculating store money
+  }
+
+  //get value from user and remember it
+  setUserValue(value) {
+    this.userValue = value;
+  }
+
+  getMoney() {
+    if (this.canGiveMoney()) {
+      this.calculateAmount();
```

Comment:
Please rename it. It isn't clear what it should calculate;

Reply:
done